osquery/sql/sqlite_util.cpp

```diff
@@ -414,6 +414,31 @@ Status QueryPlanner::applyTypes(TableColumns& columns) {
           column_types.erase(from + i);
         }
       }
+    } else if (row.at("opcode") == "Cast") {
+      auto value = boost::lexical_cast<size_t>(row.at("p1"));
```

Comment:
Will this throw an exception, if so is it handled? (If this map is guaranteed to always have these elements, there is no need to handle the exception for completeness IMO).